src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/TextCaseConverter.shared.cs

```diff
@@ -37,19 +35,20 @@ public class TextCaseConverter : ValueConverterExtension, IValueConverter
 		/// <param name="parameter">N/A</param>
 		/// <param name="culture">N/A</param>
 		/// <returns>N/A</returns>
-public object ConvertBack(object? value, Type? targetType, object? parameter, CultureInfo? culture)
+		public object ConvertBack(object? value, Type? targetType, object? parameter, CultureInfo? culture)
 			=> throw new NotImplementedException();
 
 		object? Convert(string? value, object? parameter) => GetParameter(parameter) switch
 		{
 			TextCaseType.Lower => value?.ToLowerInvariant(),
 			TextCaseType.Upper => value?.ToUpperInvariant(),
 			TextCaseType.FirstUpperRestLower when value != null && !string.IsNullOrWhiteSpace(value) => value.Substring(0, 1).ToUpperInvariant() + value.Substring(1).ToLowerInvariant(),
-			_ => value
+			_ => value?.ToString()
 		};
 
-		TextCaseType GetParameter(object? parameter) => parameter == null ? Type : parameter switch
+		TextCaseType GetParameter(object? parameter) => parameter switch
 		{
+			null => Type,
```

Comment:
I guess the default should return "Type", not just "null".
Also, please make sure Type has TextCaseType.None as default value

Reply:
@AndreiMisiukevich this isn't changing any actual behavior only simplifying this block of code as there is no point to check if `parameter` is `null` and then return `Type` otherwise go into a switch case for `parameter`...

Reply:
It seems "Type" property wasn't used before (what is wrong obviously)
And I proposed to use Type if parameter is not int or string or TextCaseType

Reply:
oh I see what you're saying... fix the default case so it's returning Type instead of TextCaseType.None... yeah that makes more sense to me as well.